webpack.config.js

```diff
@@ -142,7 +143,18 @@ module.exports = {
                     },
                     {
                         loader: 'sass-loader',
-                    }
+                    },
+                    {
+                        loader: 'sass-resources-loader',
+                            // Make PF3 and PF4 variables globably accessible to be used by the components scss
+                            options: {
+                                resources: [
+                                    path.resolve(libdir, './_global-variables.scss'),
+                                    path.resolve(nodedir, './@patternfly/patternfly/patternfly-variables.scss'),
+                                    path.resolve(nodedir, './patternfly/dist/sass/patternfly/_variables.scss')
```

Comment:
I don't think you need PF3 variables. :)

Reply:
Aren't these variables needed in lib/cockpit-components-listing-panel.scss and lib/listing.scss where variables like --pf-global--FontSize--md are used?

Reply:
Sorry too late, only the first two are needed there.